fixdep-patch.file

```diff
@@ -0,0 +1,50 @@
+diff --git a/scripts/basic/fixdep.c b/scripts/basic/fixdep.c
```

Comment:
I recommend renaming this file so that the file extension is `.patch`, for example `fixdep.patch`. If this patch is only meant for a specific version (or versions) of the kernel, may also consider elaborating the name a bit further - `fixdep-v6.8-v6.9.patch`.

Reply:
Will fix this by directly parsing the .cmd files instead of patching the source code as previously mentioned.

Reply:
https://github.com/elisa-tech/delta-kernel/issues/4